cmd/ui/src/views/Explore/GraphView.tsx

```diff
@@ -49,7 +49,7 @@ const GraphView: FC = () => {
     const theme = useTheme();
 
     const graphQuery = useSigmaExploreGraph();
-    const { data, isLoading, isError } = useAvailableEnvironments();
+    const { data, isLoading, isError } = useGraphHasData();
```

Comment:
Suggestion: rename `data` to `graphHasData` so it is a little more clear
```suggestion
    const { data:graphHasData, isLoading, isError } = useGraphHasData();
```